### DISCUSSION TITLE
摄影作品分享

### DISCUSSION THREAD

**Discussion by 01NG-VI** [share121/inter-knot#2326] [其他]
欢迎大家分享自己的摄影作品<img src="https://img-static.mihoyo.com/communityweb/upload/3831379d9e92f4050e640080e73be1bb.gif" alt="米游姬-卖萌" width="75" >
可以是绝区零
![20240717234741](https://github.com/user-attachments/assets/32859e93-106c-4f5e-a780-e854fb800ad0)
![20240717234818](https://github.com/user-attachments/assets/96ca4284-861c-49da-a7a5-39dffabfa277)
![20240717234904](https://github.com/user-attachments/assets/4cfa0292-2039-490d-b137-ca99d621f6cc)
![20240722003632](https://github.com/user-attachments/assets/c3f3e18d-d404-41a3-b944-20861cc5f009)
![20240814215438](https://github.com/user-attachments/assets/723e602a-c68d-4c4d-82d9-1ea9ff80449a)
也可以是是其他游戏（原神，星铁等）
![20230728104226](https://github.com/user-attachments/assets/15b4af33-12e1-40e8-99f9-2409a15a0417)
![20240207155855](https://github.com/user-attachments/assets/92146d7c-7c24-41e9-8e9f-136f73ce152b)
![20240809021655](https://github.com/user-attachments/assets/de6ae3b3-3f9a-4c47-b5be-f89ec29939ad)
![20240727220856](https://github.com/user-attachments/assets/8e421b14-b9b2-480a-89d5-2668bec398d7)
![20240727215306](https://github.com/user-attachments/assets/857b816f-fa86-4f2e-af22-ded456b106cb)
也可以是游戏截图
![原神 2024_7_27 21_06_16](https://github.com/user-attachments/assets/55d5b477-eea3-4ebc-b5b5-f84c92e665a8)
![崩坏：星穹铁道 2024_2_6 14_36_02](https://github.com/user-attachments/assets/48934cd1-6fe1-437a-abfd-115869a79d09)
![崩坏：星穹铁道 2024_5_8 14_29_07](https://github.com/user-attachments/assets/78110063-f6e3-4d9c-a3a8-7fda045aff80)
![MuMu12-20240812-223830](https://github.com/user-attachments/assets/ff32e85c-f52c-4059-8646-5edb3583cfd4)
![MuMu12-20240608-234415](https://github.com/user-attachments/assets/0a78ab4b-e6a9-4e3f-b3a5-4b0b3ede6252)
也可以是你拍摄的现实照片

**Comment by FYGT27** [comment]
![20240801231650](https://github.com/user-attachments/assets/55592f5b-a088-47fd-b197-22315a225211)
![Image_1722417517695](https://github.com/user-attachments/assets/32511264-9c1d-4d7e-9233-96712be92473)
画质低得嘞，权当灌水喽

**Reply by 01NG-VI** [reply]
欢迎分享<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/4b4fb49d4bd402c530600e758ebb06bd_6955490723450463851.png" alt="阿姬-开心" width="75" >

**Comment by 00190906** [comment]
好看

**Comment by KELERSAMA** [comment]
# 2023年 浙江省温州市南麂岛

虽然没有技术 但是随便一拍都很漂亮 

![image](https://github.com/user-attachments/assets/451acbf2-7aa9-4ea0-a087-57965907bdd0)

![image](https://github.com/user-attachments/assets/ffa098d2-3da2-4536-bc40-51e544b9c449)

![image](https://github.com/user-attachments/assets/489901bf-5c14-4bd7-97ff-b051b7167ca4)

![image](https://github.com/user-attachments/assets/f47ea41c-0bde-4528-b75f-510944e8904f)

# 这一张不是南麂

![image](https://github.com/user-attachments/assets/a9aa88e2-7158-4d24-8c3c-a018b1cc28e3)
